gptcache/manager/vector_data/hnswlib_store.py

```diff
@@ -0,0 +1,51 @@
+import os
+import numpy as np
+
+from gptcache.manager.vector_data.base import VectorBase, ClearStrategy
+from gptcache.utils import import_hnswlib
+
+import_hnswlib()
+
+import hnswlib  # pylint: disable=C0413
+
+
+class Hnswlib(VectorBase):
+    """vector store: hnswlib"""
+
+    def __init__(self, index_file_path: int, top_k: str, dimension: int, max_elements: int):
+        self._index_file_path = index_file_path
+        self._dimension = dimension
+        self._max_elements = max_elements
+        self._index =  hnswlib.Index(space="l2", dim=self._dimension)
+        self._top_k = top_k
+        if os.path.isfile(self._index_file_path):
+            self._index.load_index(self._index_file_path, max_elements=max_elements)
+        else:
+            self._index.init_index(max_elements=max_elements, ef_construction=100, M=16)
+        self._index.set_ef(self._top_k * 2)
```

Comment:
when `load_index`, it's not required `set_ef`.
![Uploading image.png…]()

Reply:
fixed